### ISSUE TITLE
Delegate Maintenance

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#4071]
As @josef-pkt has mentioned, there is more maintenance work than can plausible be handled by one person.

There are also a large number of PRs that are pure cleanup, affect zero logic, and require zero design decisions.  We should have a small cadre of veteran contributors enabled to handle this class of PR so jpkt can focus on more important/fun/difficult/outdoors items.

**Comment by malickf** [comment]
Hi, I'm not a veteran so I can't help with PR. But I have noticed that there are a lot of open issues that I believe can be closed. For instance :  https://github.com/statsmodels/statsmodels/issues/3829 .

If you need someone to list all these kinds of non-real issues cases I'm ready to help.

**Comment by josef-pkt** [comment]
@malickf If they have a FAQ label, then we leave them open. We don't have the wo/manpower to extract and write proper FAQs, so FAQ labeled issues is the second best thing. If FAQ issues don't have also a bug label, then they can be ignored in terms of a bug tracker.

However, help in triaging and investigating old issues would be a useful contribution. Both Kevin and I found several old issues that are outdated and were not closed when the underlying problem was fixed or disappeared, in other cases it is not clear whether the problem still exist or what the problem in the current version is. For a while in the last weeks I was just using a notebook to run examples taken from issues to see both whether there is still a problem, and if there is one how serious it is.

The only serious issues are those with a bug label, and some of the unlabeled issues that haven't been properly triaged. I recently went through many of the latter but not through all of them.

current status
187 bug https://github.com/statsmodels/statsmodels/issues?q=is%3Aopen+is%3Aissue+label%3Atype-bug
3 bug-wrong https://github.com/statsmodels/statsmodels/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20label%3Atype-bug-wrong%20
58 unlabeled https://github.com/statsmodels/statsmodels/issues?page=3&q=is%3Aopen+is%3Aissue+no%3Alabel

Some of the other labeled issues are often developer notes that are not part of a "bug-tracker", i.e. they are mostly to keep track of design and enhancement ideas or proposals.
